ui/app/src/views/import/GrafanaFlow.tsx

```diff
@@ -23,10 +23,15 @@ import { useProjectList } from '../../model/project-client';
 import { useCreateDashboardMutation } from '../../model/dashboard-client';
 import { useIsReadonly } from '../../context/Config';
 
+type Input = {
+  name: string;
+  value?: string;
+};
+
 // GrafanaLightDashboard is a Grafana dashboard that may have some variables that need to be replaced by the user.
 interface GrafanaLightDashboard {
   // The only part that is interesting us is the list of the input that can exists in the Grafana dashboard definition.
-  __inputs?: Array<{ name: string }>;
+  __inputs?: Input[];
```

Comment:
I'm not sure I get what this __inputs stands for anyway. Is this part of the grafana data model? and you say then that you make possible modification only on that part? 

You mentioned in private that maybe you should escape also the rest of the fields. Does it regards the fields below, materialized by ``[key: string]: unknown``

Reply:
Yes it's part of it when you tick the "export for sharing externally" checkbox when exporting a dashboard on Grafana. See the first PR that brought migration support for these inputs in Perses https://github.com/perses/perses/pull/816